SharedModules/universalVariables.f90

```diff
@@ -64,19 +64,21 @@ module universalVariables
                                   P_NEUTRON_MG = 2
 
   ! Search error codes
-  integer(shortInt), parameter :: valueOutsideArray = -1,&
-                                  tooManyIter       = -2,&
+  integer(shortInt), parameter :: valueOutsideArray = -1, &
+                                  tooManyIter       = -2, &
                                   targetNotFound    = -3, &
-                                  NOT_FOUND         = -3
+                                  NOT_FOUND         = -3, &

```

Comment:
why have separate flags for targetNotFound and NOT_FOUND?

Reply:
Good q

Reply:
Actually, aesthetics in the code probably!!